src/main/java/duke/command/UnmarkCommand.java

```diff
@@ -0,0 +1,38 @@
+package duke.command;
+
+import duke.task.Task;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+/**
+ * Represents an unmark command to be executed.
+ */
+public class UnmarkCommand extends Command {
+    private int index;
+
+    /**
+     * Constructor for UnmarkCommand class.
+     *
+     * @param index The TaskList index of the Task to be unmarked.
+     */
+    public UnmarkCommand(int index) {
+        super(false);
+        this.index = index;
+    }
+
+    /**
+     * Executes the list of commands to unmark a Task from a TaskList.
+     *
+     * @param taskList The given TaskList with the Task to be unmarked.
+     * @param ui The given Ui to show the status of the unmarking of the task.
+     * @param storage The given Storage that saves the TaskList locally.
+     */
```

Comment:
Perhaps you can consider combining all the commands into a class since they only consist of one method?